yarn-project/end-to-end/src/e2e_box_privatetoken.ts

```diff
@@ -0,0 +1,144 @@
+import {
```

Comment:
Maybe don't test the boxes in end-to-end. Users will have the content from the box, and should be able to run the test in there. And that's where we need to test.

Reply:
ah my concern was wanting to have the front end methods/interactions part of CI, because they have been broken before by upstream changes.  and it seemed easier to import `boxes` to test in `end-to-end` compared to trying to import some functions from `end-to-end` into `boxes`

Reply:
end-to-end is not for importing... will just copy some sandbox setup stuff over and keep tests in this subpackages